test/cases/parsing/evaluate-nullish/test.filter.js

```diff
@@ -0,0 +1,5 @@
+var supportsNullishCoalescing = require("../../../helpers/supportsNullishCoalescing");
```

Comment:
Did not know about this filter feature.. reading `test/README.md` looks useful =)